test/setuptest.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#####################################################################
+# Obtain the various ReferenceDesignator values from asset-management
+#####################################################################
+
+# set up links to the CSVs in asset-management/deployment
+ls ../../asset-management/deployment/*.csv | awk -F"/" '{print "ln -s ../../asset-management/deployment/" $NF " " $NF}' > amdeploy
+source amdeploy
+
+# extract the ReferenceDesignator column values from all these CSVs
+for i in $(ls *.csv); do
+    x=$(head -1 $i | awk -F "," '{for (x = 1; x <= NF; ++x) if ($x == "Reference Designator") print x}')
+    awk -F "," '{print $refdes}' refdes=$x $i | grep -v "Reference Designator" > ${i}.refdes
+done
+
+# get all the unique ReferenceDesignator values
+sort -u *csv.refdes > refdes.txt
+
+# remove all the CSV links
+find . -type l -name "*.csv" -exec unlink {} \;
+
+# cleanup temp files
+rm *.refdes amdeploy
+
+##################################################################
+# Obtain the units, types and parameters from preload-database.sql
+##################################################################
+
+# set up a link to preload-database/preload-database.sql
```

Comment:
Extracting data from the preload database should be done in python test setup, not in the shell script.

Reply:
That's been done